tools/wptrunner/wptrunner/executors/platformaccessibility/ia2/constants.py

```diff
@@ -0,0 +1,346 @@
+    IA2_ROLE_DESKTOP_PANE: "IA2_ROLE_DESKTOP_PANE",  # noqa: F821
+    IA2_ROLE_DIRECTORY_PANE: "IA2_ROLE_DIRECTORY_PANE",  # noqa: F821
+    IA2_ROLE_EDITBAR: "IA2_ROLE_EDITBAR",  # noqa: F821
+    IA2_ROLE_EMBEDDED_OBJECT: "IA2_ROLE_EMBEDDED_OBJECT",  # noqa: F821
+    IA2_ROLE_ENDNOTE: "IA2_ROLE_ENDNOTE",  # noqa: F821
+    IA2_ROLE_FILE_CHOOSER: "IA2_ROLE_FILE_CHOOSER",  # noqa: F821
+    IA2_ROLE_FONT_CHOOSER: "IA2_ROLE_FONT_CHOOSER",  # noqa: F821
+    IA2_ROLE_FOOTER: "IA2_ROLE_FOOTER",  # noqa: F821
+    IA2_ROLE_FOOTNOTE: "IA2_ROLE_FOOTNOTE",  # noqa: F821
+    IA2_ROLE_FORM: "IA2_ROLE_FORM",  # noqa: F821
+    IA2_ROLE_FRAME: "IA2_ROLE_FRAME",  # noqa: F821
+    IA2_ROLE_GLASS_PANE: "IA2_ROLE_GLASS_PANE",  # noqa: F821
+    IA2_ROLE_HEADER: "IA2_ROLE_HEADER",  # noqa: F821
+    IA2_ROLE_HEADING: "IA2_ROLE_HEADING",  # noqa: F821
+    IA2_ROLE_ICON: "IA2_ROLE_ICON",  # noqa: F821
+    IA2_ROLE_IMAGE_MAP: "IA2_ROLE_IMAGE_MAP",  # noqa: F821
+    IA2_ROLE_INPUT_METHOD_WINDOW: "IA2_ROLE_INPUT_METHOD_WINDOW",  # noqa: F821
+    IA2_ROLE_INTERNAL_FRAME: "IA2_ROLE_INTERNAL_FRAME",  # noqa: F821
+    IA2_ROLE_LABEL: "IA2_ROLE_LABEL",  # noqa: F821
+    IA2_ROLE_LAYERED_PANE: "IA2_ROLE_LAYERED_PANE",  # noqa: F821
+    IA2_ROLE_NOTE: "IA2_ROLE_NOTE",  # noqa: F821
+    IA2_ROLE_OPTION_PANE: "IA2_ROLE_OPTION_PANE",  # noqa: F821
+    IA2_ROLE_PAGE: "IA2_ROLE_PAGE",  # noqa: F821
+    IA2_ROLE_PARAGRAPH: "IA2_ROLE_PARAGRAPH",  # noqa: F821
+    IA2_ROLE_RADIO_MENU_ITEM: "IA2_ROLE_RADIO_MENU_ITEM",  # noqa: F821
+    IA2_ROLE_REDUNDANT_OBJECT: "IA2_ROLE_REDUNDANT_OBJECT",  # noqa: F821
+    IA2_ROLE_ROOT_PANE: "IA2_ROLE_ROOT_PANE",  # noqa: F821
+    IA2_ROLE_RULER: "IA2_ROLE_RULER",  # noqa: F821
+    IA2_ROLE_SCROLL_PANE: "IA2_ROLE_SCROLL_PANE",  # noqa: F821
+    IA2_ROLE_SECTION: "IA2_ROLE_SECTION",  # noqa: F821
+    IA2_ROLE_SHAPE: "IA2_ROLE_SHAPE",  # noqa: F821
+    IA2_ROLE_SPLIT_PANE: "IA2_ROLE_SPLIT_PANE",  # noqa: F821
+    IA2_ROLE_TEAR_OFF_MENU: "IA2_ROLE_TEAR_OFF_MENU",  # noqa: F821
+    IA2_ROLE_TERMINAL: "IA2_ROLE_TERMINAL",  # noqa: F821
+    IA2_ROLE_TEXT_FRAME: "IA2_ROLE_TEXT_FRAME",  # noqa: F821
+    IA2_ROLE_TOGGLE_BUTTON: "IA2_ROLE_TOGGLE_BUTTON",  # noqa: F821
+    IA2_ROLE_UNKNOWN: "IA2_ROLE_UNKNOWN",  # noqa: F821
+    IA2_ROLE_VIEW_PORT: "IA2_ROLE_VIEW_PORT",  # noqa: F821
+    IA2_ROLE_COMPLEMENTARY_CONTENT: "IA2_ROLE_COMPLEMENTARY_CONTENT",  # noqa: F821
+    IA2_ROLE_LANDMARK: "IA2_ROLE_LANDMARK",  # noqa: F821
+    IA2_ROLE_LEVEL_BAR: "IA2_ROLE_LEVEL_BAR",  # noqa: F821
+    IA2_ROLE_CONTENT_DELETION: "IA2_ROLE_CONTENT_DELETION",  # noqa: F821
+    IA2_ROLE_CONTENT_INSERTION: "IA2_ROLE_CONTENT_INSERTION",  # noqa: F821
+    IA2_ROLE_BLOCK_QUOTE: "IA2_ROLE_BLOCK_QUOTE",  # noqa: F821
+    IA2_ROLE_MARK: "IA2_ROLE_MARK",  # noqa: F821
+    IA2_ROLE_SUGGESTION: "IA2_ROLE_SUGGESTION",  # noqa: F821
+    IA2_ROLE_COMMENT: "IA2_ROLE_COMMENT",  # noqa: F821
+}
+
+STATE_SYSTEM_UNAVAILABLE = 0x00000001

```

Comment:
Similarly, maybe we can use `enum.Flag` here?

```py
class State(enum.IntFlag):
    STATE_SYSTEM_UNAVAILABLE = 0x00000001
```

... so that `get_msaa_state_list()` can just be:

```py
def get_msaa_state_list(states: State):
    return [state.name.removeprefix('STATE_SYSTEM_') for state in states]
```

... or something like that.